wordpress-mobile/WordPress-Android
# Import latest login library (do not merge)

Imports the updates to the login library made from the WooCommerce app, with some WPAndroid changes to accommodate them.

**This PR should not be merged** - once the reviewer is happy with this and the corresponding login library PR (https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/6), the login library PR should be merged and I'll open a new WPAndroid PR pulling in the changes from `develop` (instead of from the feature branch this PR is using).

[This view](https://github.com/wordpress-mobile/WordPress-Android/compare/121154d...cbe1c63) of only the changes to WPAndroid may be helpful.

To test:
* A general pass through the login flows, mostly for visual bugs
* Magic links should open the WordPress app (or offer a choice between the stable and dev versions, if both are installed)
